opentelemetry-otlp/src/exporter/tonic.rs

```diff
@@ -25,11 +26,31 @@ pub struct TonicConfig {
 ///
 /// [tonic]: <https://github.com/hyperium/tonic>
 /// [channel]: tonic::transport::Channel
-#[derive(Default, Debug)]
+#[derive(Default)]
 pub struct TonicExporterBuilder {
     pub(crate) exporter_config: ExportConfig,
     pub(crate) tonic_config: TonicConfig,
     pub(crate) channel: Option<tonic::transport::Channel>,
+    pub(crate) interceptor: Option<BoxInterceptor>,
+}
+
+pub(crate) struct BoxInterceptor(Box<dyn tonic::service::Interceptor>);
```

Comment:
This bound should be on the interceptor impl rather than assumed via unsafe.

```suggestion
pub(crate) struct BoxInterceptor(Box<dyn tonic::service::Interceptor + Send>);
```